src/unistd.rs

```diff
@@ -1439,6 +1439,32 @@ pub fn sleep(seconds: c_uint) -> c_uint {
     unsafe { libc::sleep(seconds) }
 }
 
+#[cfg(target_family = "unix")]
```

Comment:
The `target_family` should be unnecessary.  Nix only runs on unix operating systems.